src/lib/autoinstall/script.rb

```diff
@@ -408,6 +421,60 @@ def script_path
     end
   end
 
-  # List of known script
+  # Scripts that are used when processing the <ask-list> section
+  class AskScript < Y2Autoinstallation::ExecutedScript
+    attr_reader :environment
+
+    def self.type
+      "ask-scripts"
+    end
+
+    def initialize(hash)
+      super
+      @environment = !!hash["environment"]
+    end
+
+    def to_hash
+      super.merge("environment" => environment)
+    end
+
+    # @see Y2Autoinstallation::Script
+    def logs_dir
+      File.join(Yast::AutoinstConfig.tmpDir, self.class.type, "logs")
```

Comment:
is it logged before? What if it contain sensitive information?

Reply:
Yes, this one is logged (as any other script). The `AskDefaultValueScript` is not exactly for that reason.